app/js/containers/Seller/4_SellerMarginContainer.jsx

```diff
@@ -5,6 +5,10 @@ import {connect} from 'react-redux';
 import MarginSelectorForm from '../../components/Seller/MarginSelectorForm';
 import Loading from '../../components/ui/Loading';
 import newSeller from "../../features/newSeller";
+import network from '../../features/network';
+import prices from '../../features/prices';
+import {getPrices} from '../../features/prices/reducer';
```

Comment:
You should not be able to access internal element of each feature, that should exposed through the index.
You can see example for others